using-babelfish/getting-started.md

```diff
@@ -22,19 +22,26 @@ The easiest way to run the _bblfshd_ daemon is using Docker. You can start it wi
 $ docker run -d --name bblfshd --privileged -p 9432:9432 bblfsh/bblfshd
 ```
 
-This will run the image in a stateless mode, meaning that any installed drivers will be lost when you stop the container. To avoid this from happening, add the `-v` parameter to keep the `/var/lib/bblfshd` directory in a volume:
+This will run the image in a stateless mode, meaning that any installed drivers will be lost when you stop the container. To avoid this from happening add the `-v` parameter to keep the container internal `/var/lib/bblfshd` directory in a volume.
```

Comment:
The container internal filesystem (containers "contain" a mini system inside), would "container internal filesystem" be clearer?

Reply:
Yes, I think so. What about: 
- To avoid this from happening add the `-v` parameter to keep the container internal filesystem in a volume at `/var/lib/bblfshd` directory.

or 
- To avoid this from happening add the `-v` parameter to keep the container internal filesystem in a volume at `/var/lib/bblfshd`.

Reply:
What about simplifiying?

"To avoid this from happening, add the -v parameter to keep the container internal filesystem at a volume".

Reply:
"To avoid this from happening, add the -v parameter in the above command, to keep the container internal filesystem at a volume". Maybe?

Reply:
Yes, sorry, should have skipped the notification, I like your version better.

Reply:
Done, are you ok with merging this?

Reply:
Perfect, thanks!